### ISSUE TITLE
Turn old multi-image helpers (sequenceMode, collectionMode, reference strip) into plugins

### ISSUE THREAD

**Issue by iangilman** [openseadragon/openseadragon#649]
When they were created originally, they were hacks to allow a little bit of multi-image-ness. Now that we have the full range of multi-image possibilities, they would probably be better as plugins, opening up the field to many other related features as plugins.

**Comment by avandecreme** [comment]
:+1:

**Comment by iangilman** [comment]
Note that @atacraft has expressed interest in doing the sequence mode plugin over in #761.

**Comment by phivk** [comment]
I am currently looking into customizing the position of the `Image Reference Strip` and would benefit from more customizability options. Defining this component as a plugin might indeed help with that.

@iangilman Are there any updates since this issue was last updated? Or is targeting `div.referencestrip` and surrounding elements with CSS the way to go?

**Comment by iangilman** [comment]
There have not been any updates... I'd still love for someone to turn it into a plugin, but no one has yet. 

Targeting it with CSS is a good start. You can also take a look at the reference strip code... There's not a lot to it.

**Comment by phivk** [comment]
I did have a look at the reference strip code and noticed something odd: there is the `referenceStripPosition` option that [get's assigned](https://github.com/openseadragon/openseadragon/blob/31bee4028f44a9ad7864d1734856ed0e9acf55d0/src/referencestrip.js#L84) in the reference strip constructor, but seems then ignored [when the strip is actually positioned](https://github.com/openseadragon/openseadragon/blob/31bee4028f44a9ad7864d1734856ed0e9acf55d0/src/referencestrip.js#L132-L140)...

@iangilman did I assess that correctly?

**Comment by iangilman** [comment]
Yes, looks like you're correct. Unfortunately the reference strip code hasn't had a lot of love.

**Comment by iangilman** [comment]
One thing to be aware of with the reference strip is that its prototype has been extended with `OpenSeadragon.Viewer`, so it inherits a lot from that. I don't think `position` is used in that code though.
